src/core.js

```diff
@@ -1386,6 +1386,16 @@ $.extend( $.validator, {
 
 		// https://jqueryvalidation.org/date-method/
 		date: function( value, element ) {
+			if ( this.settings.debug && window.console ) {
```

Comment:
should we warn only on the first occurence? Doing it all the time could get a perf problem?

Reply:
Good point. I'll update it shortly.